core/common/base.py

```diff
@@ -74,47 +75,47 @@ def receive_data(self, size: int = 4096, data_type: str = str) -> Optional[Any]:
 
 class Server(BaseSocketServer):
     """
-    A class used to abstract the socket that will act as the server
+    A class used to abstract the socket that will act as the server.
 
     ....
     """
 
     def __init__(self, ip, port):
-        """
+        """Initialize Server Object with ip and port parameters.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)